src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java

```diff
@@ -13,4 +18,19 @@ public class RespondentLiPResponse {
 
     private MediationLiP respondent1MediationLiPResponse;
     private FinancialDetailsLiP respondent1LiPFinancialDetails;
+    private String respondent1ResponseLanguage;
+
+    @JsonIgnore
+    public boolean isRespondentResponseBilingual(CaseData caseData) {
```

Comment:
my suggestion in here is:
    public boolean isRespondentResponseBilingual(CaseData caseData) {
        boolean responseLanguage = Optional.ofNullable(caseData.getCaseDataLiP())
            .map(CaseDataLiP::getRespondent1LiPResponse)
            .map(RespondentLiPResponse::getRespondent1ResponseLanguage)
            .filter(respondent1ResponseLanguage -> Language.BOTH.toString().equals(respondent1ResponseLanguage))
            .isPresent();

        return responseLanguage;
    }

what do you think about it?